CppCoreGuidelines.md

```diff
@@ -1,14 +1,14 @@
 # <a name="main"></a>C++ Core Guidelines
 
-January 22, 2018
+January 25, 2018
 
 
 Editors:
 
 * [Bjarne Stroustrup](http://www.stroustrup.com)
 * [Herb Sutter](http://herbsutter.com/)
 
-This document is an early draft. It's known to be incomplet, incorrekt, and has lots of b**a**d **for**~mat~ting.
+This document is an early draft. It's known to be incomplete, incorrekt, and has lots of b**a**d **for**~mat~ting.
```

Comment:
this "incomplet" is intentional

Reply:
Thank you for calling attention!
Reverted - sorry.